tests/plugins/test_koji_promote.py

```diff
@@ -737,6 +741,10 @@ def validate_output(self, output, metadata_only):
                 digest_pullspec = image.to_str(tag=False) + '@' + fake_digest(image)
                 assert digest_pullspec in repositories_digest
 
+            tags = docker['tags']
+            assert isinstance(tags, list)
+            assert all(is_string_type(tag) for tag in tags)
```

Comment:
I got nothing... I was thinking we could extract the tag from each image, but this seems wrong for a test. The expected values should be hard coded and not dynamically generated. Perhaps this validation method can take a list of expected tags? This is very minor as the `test_koji_promote_image_tags` already test the code changes.